README.md

```diff
@@ -68,10 +69,18 @@ If you want to set up a cronjob, send the output to a temp file and when finishe
 file over the "actual" file. Otherwise you might end up with an incomplete page when the checks are 
 running. Like so:
 
-	* * * * * /bin/bash /opt/srvmon/srvmon.sh > /var/www/index.html.tmp && /bin/mv /var/www/index.html.tmp /var/www/index.html
+	* * * * * /bin/bash /opt/srvmon/srvmon > /var/www/index.html.tmp && /bin/mv /var/www/index.html.tmp /var/www/index.html
 
 If the check fails for whatever reason, the "old" page will not be overridden.
 
+### CGI mode
+
+If you want to set up CGI mode, you need to copy the script to your server CGI directory. 
+You can use `docker` for this purpose. Like so:
+
```

Comment:
Could you also please add instructions for usage without docker?

Reply:
That's the above line. If you already have CGI enabled, you kinda just drop the script in a directory for it to work.

Reply:
Okay thanks for the clarification. I've never used cgi scripts, a bit before my time